src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -288,6 +296,41 @@ private function registerDoctrineExtensionConfiguration(ContainerBuilder $contai
         }
     }
 
+    private function registerHttpCache(ContainerBuilder $container, array $config, XmlFileLoader $loader)
+    {
+        $loader->load('http_cache.xml');
+
+        if (true !== $config['http_cache']['enable_tags']) {
```

Comment:
When `enable_tags` is true and `maxAge` > 0. We should warn the developer that the browser will cache responses and remove the benefits of cache invalidation.